websites/K/KEXP/presence.ts

```diff
@@ -0,0 +1,67 @@
+const presence = new Presence({ clientId: "1138970637440917504" }),
+	supportedPaths = [
+		"/archive/",
+		"/podcasts/",
+		"/shows/",
+		"/djs/",
+		"/schedule/",
+	],
+	timeElapsed = Math.floor(Date.now() / 1000);
+async function updatePresence() {
+	const strings = await presence.getStrings({
+			pause: "general.paused",
+			play: "general.playing",
+			profile: "general.readingAbout",
+			reading: "general.readingAnArticle",
+			watching: "general.watching",
+		}),
+		currentPagePath = window.location.pathname,
+		presenceData: PresenceData = {
+			startTimestamp: timeElapsed,
+			largeImageKey: "https://i.imgur.com/uzXz274.png",
+			buttons: [{ label: "Check KEXP", url: "https://kexp.org" }],
+		};
+
+	function setDetails(detailsKey: keyof typeof strings, imageKey: string) {
+		presenceData.details = strings[detailsKey];
+		presenceData.smallImageKey = imageKey;
+		presenceData.smallImageText = strings[detailsKey];
+	}
+
+	if (supportedPaths.includes(currentPagePath)) {
+		setDetails("watching", Assets.Viewing);
+		presenceData.state = document.querySelector(".InlineMenu-current");
+	} else if (
+		currentPagePath.startsWith("/podcasts/") ||
+		currentPagePath.startsWith("/shows/")
+	) {
+		setDetails("reading", Assets.Reading);
+		presenceData.state = document.querySelector("h1")?.textContent;
+	} else if (currentPagePath.startsWith("/djs/")) {
+		setDetails("profile", Assets.Reading);
+		presenceData.state = document.querySelector("h3")?.textContent;
+	} else {
+		const coverImage = document
+			.querySelector(".Player-coverImage")
+			?.getAttribute("src");
+
+		presenceData.details = document.querySelector("a.Player-show")?.textContent;
+		presenceData.largeImageKey = coverImage?.startsWith("https://")
+			? coverImage
+			: "https://i.imgur.com/RBf6mDU.png";
```

Comment:
Use the square image. Alternatively, use an if statement to conditionally set the `largeImageKey`
```suggestion
			: "https://i.imgur.com/uzXz274.png";
```